_includes/manuals/nightly/3.4_install_from_source.md

```diff
@@ -1,81 +1 @@
-
-Foreman will run with the following requirements (aside from rubygem dependencies):
-
-* Ruby 2.5 or newer
-* NodeJS 10 or newer
-* NPM 3.0 or newer
-
-You will want to make sure that you have *postgresql-devel* installed so the database
-gems can install properly. Also, you would also need *gcc*, *ruby-devel*,
-*libxml-devel*, *libxslt-devel*, *libvirt-devel*, *nodejs*, and *npm* packages.
-
-For RHEL7 or clones, you can issue the following commands to install *all*
-required packages:
-
-    yum groupinstall "Development Tools" "Development Libraries"
-    yum -y install gcc-c++ git ruby ruby-devel rubygems \
-        libvirt-devel postgresql-devel openssl-devel \
-        libxml2-devel libxslt-devel zlib-devel \
-        readline-devel systemd-devel tar nodejs npm libcurl-devel
-
-Additionally, it is important that `config/database.yml` is set to use
-the correct database in the "production" block. Rails (and subsequently
-Foreman) will use these connection settings under "production" to manage
-the database it uses and setup the necessary schema.
-
-{% highlight bash %}
-git clone https://github.com/theforeman/foreman.git -b develop
-cd foreman
-cp config/settings.yaml.example config/settings.yaml
-cp config/database.yml.example config/database.yml
-gem install bundler
-bundle install --without development test --path vendor
-npm install
-# set up database schema, precompile assets and locales
-RAILS_ENV=production bundle exec rake db:migrate
-RAILS_ENV=production bundle exec rake db:seed assets:precompile locale:pack webpack:compile
-{% endhighlight %}
-
-The db:seed step will print out the default admin password, record this in order to log in later.
-
-In order to run Foreman you can use the following command inside your git repository:
-
-{% highlight bash %}
-./bin/rails s -e production
-{% endhighlight %}
-
-<div class="alert alert-info">This manual will recommend <code>foreman-rake &lt;task&gt;</code> to run rake tasks, however when installed from source, replace this with <code>bundle exec rake &lt;task&gt; RAILS_ENV=production</code></div>
-
-### Latest stable version
-
-to get latest "stable" version do:
-
-{% highlight bash %}
-git clone https://github.com/theforeman/foreman.git -b {{page.version}}-stable
-{% endhighlight %}
-
-### CLI (Hammer)
-
-To install hammer from git checkouts, you will just need ```rake``` installed on your system.
-Clone and install CLI core
-
-{% highlight bash %}
-$ git clone https://github.com/theforeman/hammer-cli.git
-$ cd hammer-cli
-$ rake install
-$ cd ..
-{% endhighlight %}
-
-and clone plugin with foreman commands
-
-{% highlight bash %}
-$ git clone https://github.com/theforeman/hammer-cli-foreman.git
-$ cd hammer-cli-foreman
-$ rake install
-$ cd ..
-{% endhighlight %}
-
-You can install other hammer plugins via any of the methods mentioned above.
+Installation from sources was moved to [contribute](/contribute.html) page.
```

Comment:
So we are saying this method is still supported, we just moved the docs?

Reply:
Hmm I can probably state here that this is meant for development setups.

Reply:
I'm good with that change and thus stating it here. @tbrisker ?

Reply:
Makes sense to me to mention this isn't supported for production setups.